openapi/saps4hana.itcm.customerhierarchy/Ballerina.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+license = ["Apache-2.0"]
+keywords = ["sap", "saps4hana", "customerhierarchy", "paid"]
```

Comment:
You can add a space for key words
"customer hierarchy"

Reply:
Fixed this